client/lib/components/home_page_sections/home_page_recent_numbers.dart

```diff
@@ -96,7 +96,9 @@ class __HomeStatsFaderState extends State<_HomeStatsFader>
   Widget build(BuildContext context) {
     final countryList = Provider.of<IsoCountryList>(context);
     final currentCountryCode = widget.statsStore.countryIsoCode;
-    final country = countryList.countries[currentCountryCode!];
+    var country;
+    if (currentCountryCode != null)
+      country = countryList.countries[currentCountryCode];
```

Comment:
I was thinking you could simplify this with [Conditional Property Access](https://dart.dev/codelabs/dart-cheatsheet#conditional-property-access)... then I realize that doesn't work and you need to do it this way instead ;-)

Reply:
Ah good call on simplifying this bit! After playing around some more, I think we can just do:

`final country = countryList.countries[currentCountryCode];`

(sorry, I could've sworn this was causing a runtime error but it's working fine for me now 😛)